### PR TITLE
Feature: Request events data from API and display in Home page

### PR DESCRIPTION
**Pull request by SilviaTrujillano23**
## Describe your changes

- Replace firebase method for axios method(get) 
- Display events in Home, requesting data from backend

## Issue ticket number and link
https://trello.com/c/o5hRUXy3

## Checklist before requesting a review
- [x] I have performed a self-review of my code
- [x] If it is a core feature, I have added thorough tests.
- [ ] Do we need to implement analytics?
- [ ] Will this be part of a product update? If yes, please write one phrase about this update: 
- [ ] Create storybook for components involved

Homepage:
image

### CODE REVIEW

File: src/shared/api/endpoints/conferences.ts [hunk 1]
```diff
@@ -1,13 +1,15 @@
-import { getAppCollections } from './base'
+import { requests } from '../../../api/baseRequest'
 
 import { Conference } from './../../entities'
 
 function ConferenceAPI() {
-  const collectionName = 'events'
```
Comment (anversa-pro) [inline src/shared/api/endpoints/conferences.ts:6]: Please move this hard code to a constants file for reuse

File: src/shared/api/endpoints/conferences.ts [hunk 2]
```diff
@@ -1,13 +1,15 @@
...
-
   const getAll = async (): Promise<Conference[]> => {
-    const events = await getAppCollections(collectionName)
-    const conferences: Conference[] = events.map(event => (event as Conference))
+    const { data: myData } = await requests.get(`events`)
+    console.log(myData)
```
Comment (anversa-pro) [inline src/shared/api/endpoints/conferences.ts:8]: Here you also can use the constant name

File: src/shared/api/endpoints/conferences.spec.ts [hunk 3]
```diff
@@ -2,9 +2,13 @@ import { Conference } from '../../entities'
 
 import ConferenceAPI from './conferences'
 
-const mockFn = jest.fn()
+/* const mockFn = jest.fn()
 jest.mock('./base', () => ({
   getAppCollections: () => mockFn(),
+})) */
```
Comment (jlora-mck) [inline src/shared/api/endpoints/conferences.spec.ts:8]: borrar!

File: src/shared/api/endpoints/conferences.ts [hunk 4]
```diff
@@ -1,13 +1,15 @@
...
+
+    const conferences: Conference[] = myData.data.map(
```
Comment (jlora-mck) [inline src/shared/api/endpoints/conferences.ts:10]:
```suggestion
    const  { data: eventData } = myData;
    const conferences: Conference[] = eventDatamap(
```

**Review by anversa-pro — APPROVED** [review]

**Review by jlora-mck — APPROVED** [review]

**Review by FelipeLosada1996 — APPROVED** [review]

File: src/shared/api/endpoints/conferences.ts [hunk 5]
```diff
@@ -1,13 +1,15 @@
...
-  const collectionName = 'events'
-
   const getAll = async (): Promise<Conference[]> => {
-    const events = await getAppCollections(collectionName)
-    const conferences: Conference[] = events.map(event => (event as Conference))
+    const { data: myData } = await requests.get(config.eventCollectionName)
+
+    const { data: eventData } = myData
+    const conferences: Conference[] = eventData.map(
+      (event: any) => event as Conference
```
Comment (jose-sal-y-rosas) [inline src/shared/api/endpoints/conferences.ts:11]: Please considering avoid using any. Eslint will show an error here.

**Review by jose-sal-y-rosas — APPROVED** [review]
👍🏼

**Pull request merged**